kanidmd/idm/src/server.rs

```diff
@@ -2656,21 +2667,23 @@ impl<'a> QueryServerWriteTransaction<'a> {
     }
 
     fn reload_domain_info(&mut self) -> Result<(), OperationError> {
-        let domain_name = self.get_db_domain_name()?;
-
-        let mut_d_info = self.d_info.get_mut();
-        if mut_d_info.d_name != domain_name {
-            admin_warn!(
-                "Using database configured domain name {} - was {}",
-                domain_name,
-                mut_d_info.d_name,
-            );
-            admin_warn!(
-                "If you think this is an error, see https://kanidm.github.io/kanidm/administrivia.html#rename-the-domain"
-            );
-            mut_d_info.d_name = domain_name;
-        }
-        Ok(())
+        spanned!("server::reload_domain_info", {
+            let domain_name = self.get_db_domain_name()?;
+
+            let mut_d_info = self.d_info.get_mut();
+            if mut_d_info.d_name != domain_name {
+                admin_warn!(
+                    "Using database configured domain name {} - was {}",
+                    domain_name,
+                    mut_d_info.d_name,
+                );
+                admin_warn!(
+                    "If you think this is an error, see https://kanidm.github.io/kanidm/administrivia.html#rename-the-domain"
```

Comment:
This link needs updating to https://kanidm.github.io/kanidm/master/administrivia.html#rename-the-domain

Reply:
Probably should be stable actually?  But yes I'll update.